resources/views/inc/home_footer.blade.php

```diff
@@ -2,7 +2,7 @@
     <div class="container bg-secondary">
         <div id="partnerContainer" class="footer-social bg-secondary">
             <h5>Visit our partners:</h5>
-            <a href="http://www.vatstar.com/" target="_blank"><img src="/photos/partner-vatstar.png" alt="VATSTAR Training"></a><a id="teamspeak_logo" href="https://www.teamspeak.com/?tour=yes" target="_blank"><img src="/photos/partner-ts.png" alt="TeamSpeak"></a><a href="https://www.thepilotclub.org/" target="_blank"><img src="/photos/partner-tpc.png" alt="The Pilot Club"></a>
```

Comment:
Can we also remove this photo from the files?

Reply:
@iccowan Done - thanks for the catch on that one.